experimental/add_to_app/android_fullscreen/app/build.gradle

```diff
@@ -5,35 +5,37 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileOptions {
-        sourceCompatibility 1.8
-        targetCompatibility 1.8
-    }
     compileSdkVersion 28
+    buildToolsVersion "29.0.2"
     defaultConfig {
-        applicationId "com.example.simpleandroidapp"
+        applicationId "dev.flutter.example.androidfullscreen"
         minSdkVersion 19
         targetSdkVersion 28
         versionCode 1
         versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
```

Comment:
@blasten, as of like today, we don't need this anymore right?

Reply:
I don't think so. was this created by Android Studio?

Reply:
Closing since this comment appears to have been added to the wrong line.